inductiva/templates/protein_solvation/gromacs/commands.json

```diff
@@ -0,0 +1,9 @@
+[{"method_name": "pdb2gmx", "f": "protein.pdb", "o": "protein.gro", "water": "tip3p", "user_input": "6"},
```

Comment:
I am guessing that @sarmento would suggest that here we keep things even more "barebones" without using any of our abstractions for commands, and just list the raw command lines, such that it is a trivial "copy-paste" from GROMACS tutorials, etc and there is nothing Inductiva-specific here. 

Note: this might seem to go against the direction we were following to support security validation of the commands (against code injection for example), but maybe we can still achieve that to some extent while living the json file very clean and simple?

Reply:
I agree that this goes a bit against what has been already implemented by David to ensure that there isn't code injection.

Reply:
But I agree that that solution is more user friendly.

Reply:
I think here the focus is the scenarios and thus users won't need to copy-paste anything. For developing new scenarios, this does not seem like a large overhead to implement and the .json file seems clean enough to understand. 

In my view, this implementation even simplifies further the executers implementation and can be made more generic for other simulators. Hence, the deployment of new simulators will be super easy!

The "barebone" implementation would require a new refactor on the backend to accomodate this and the security validation, which seems that would be going a few steps backward. Notice that, knowledgable users could change the commands.json file if they want and inject code on the executers, unless we save it on the backend. The latter will be an extra thing to add on the backend for each new scenario.

Moreover, this allows for a sequence of commands to be run on a low-level simulation (fixing also the concept of Pipe we were discussing on the sync) and the security of those commands is still validated by us before running them.

Reply:
I agree with @IvanPombo. My suggestion is to implement Fabio's changes, as this solution already solves the issue of executing the list of commands and unblocks me. Afterwards, we can discuss a more "barebones" solution, which would require bigger modifications to the backend.